packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-number-list.tsx

```diff
@@ -1,16 +1,18 @@
 import * as React from 'react'
+import cx from 'classnames'
 import { TeamsProcessedSvgIconSpec } from '../types'
+import { teamsIconSlotClassNames } from '../teamsIconSlotClassNames'
 
 export default {
   icon: ({ classes }) => (
     <svg role="presentation" focusable="false" viewBox="8 8 16 16" className={classes.svg}>
       <g>
         <path
-          className={classes.outlinePart}
+          className={cx(teamsIconSlotClassNames.outline, classes.outlinePart)}
```

Comment:
don't like the fact that this class name data is repeated for every icon while, in fact, it provides the same information as `classes.outlinePart`. We might think about some way to provide `classes` via styles, via dedicated prop - so that this repetition will be avoided. Note that this will be quite safe move (as it is quite easy to filter out this prop before Fela transformations), as well as we will still preserve full control of this aspect being provided to the client's hands (so it will be up to client to decide on class names policy, extend this set, etc)